lib/models/seed.dart

```diff
@@ -0,0 +1,15 @@
+class SeedModel {
+  String seed;
+  DateTime expiresAt;
+
+  SeedModel({required this.seed, required this.expiresAt});
```

Comment:
**fix**: please make your instance members `final` so that you can use a `const` constructor here